ui/apps/dashboard/src/components/Navigation/Environments.tsx

```diff
@@ -0,0 +1,232 @@
+'use client';
+
+import { Fragment, useState } from 'react';
```

Comment:
Envs nav menu is mostly a copy paste of the old env select menu with new style IA applied.